golang/vaas/pkg/options/options.go

```diff
@@ -0,0 +1,14 @@
+package options
```

Comment:
Why don't you move this into the `vaas` package? I only see vaas related options in this package :D

Reply:
I'm using it also in the `message` package